manifest.ini

```diff
@@ -1,6 +1,18 @@
 [stage]
 server = https://kinto.stage.mozaws.net/v1
 writer_server = https://kinto-writer.stage.mozaws.net/v1
+we_server_url = https://webextensions-settings.stage.mozaws.net/v1
+account_server_url = https://api.accounts.firefox.com/v1
+oauth_server_url = https://oauth.accounts.firefox.com/v1
+fxa_user_email =
+fxa_user_password =
+qa_collection = 3e0f013d2362aa4fa7b0db2fefb40011b9545d78dcca7aabc0d32f0b42f1733d
 
 [prod]
-server = https://firefox.settings.services.mozilla.com/v1/ 
+server = https://firefox.settings.services.mozilla.com/v1/
+we_server_url = https://webextensions.settings.services.mozilla.com/v1
+account_server_url = https://api.accounts.firefox.com/v1
+oauth_server_url = https://oauth.accounts.firefox.com/v1
+fxa_user_email =
+fxa_user_password =
+qa_collection = 3e0f013d2362aa4fa7b0db2fefb40011b9545d78dcca7aabc0d32f0b42f1733d
```

Comment:
Do we want the `qa_collection` uuid to be the same between stage and prod, or do we care?